### ISSUE TITLE
Python 3 ImportError

### ISSUE THREAD

**Issue by stsievert** [raamana/pyradigm#6]
I installed with `pip install pyradigm`. Then with Python 3.6,

``` python
In [2]: import pyradigm
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-2-1e91a6f90dae> in <module>()
----> 1 import pyradigm

/Users/scott/anaconda/lib/python3.6/site-packages/pyradigm/__init__.py in <module>()
      1
----> 2 from pyradigm import MLDataset
      3
      4 __all__ = [ 'pyradigm', 'MLDataset' ]
      5

ImportError: cannot import name 'MLDataset'
```

But under Python 2.7:

``` python
In [1]: from pyradigm import MLDataset

In [2]:
```

**Comment by raamana** [comment]
Thanks Scott, I will improve the compatibility with py3+ soon, but should this hold back the acceptance? I think Ariel clarified it was encouraged, but not necessary.

**Comment by raamana** [comment]
Python 3.5, 3.6 and 2.7 are all supported now - with 2c35f1154976af7dd40f1a4914ebc330e4f75cb2.

check https://travis-ci.org/raamana/pyradigm

Reopen if something is amiss.

**Comment by stsievert** [comment]
Python 3 support is not required, but if it doesn't install under Python 3 there should be a note in the readme.

In __init__.py I don't think you need to specify exactly 2.7.13. 2.7 is good enough.

**Comment by raamana** [comment]
yes, 2.7 might be enough, but I wanted to be specific as to what was being tested for support.

**Comment by stsievert** [comment]
Can you be specific in other places, and not enforce that the version must be 2.7.13? 

> the revision number is incremented when minor bugs are fixed. https://en.m.wikipedia.org/wiki/Software_versioning

So 2.7.13 should only have minor bug fixes, which is true according to the [2.7.13 changelog][1].

[1]:https://hg.python.org/cpython/raw-file/v2.7.13/Misc/NEWS
